com.ca.lsp.cobol/lsp-core-cobol-parser/src/test/java/com/ca/lsp/core/cobol/visitor/LevenshteinDistanceTest.java

```diff
@@ -0,0 +1,101 @@
+/*
+ *
+ *  * Copyright (c) 2020 Broadcom.
+ *  * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *  *
+ *  * This program and the accompanying materials are made
+ *  * available under the terms of the Eclipse Public License 2.0
+ *  * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *  *
+ *  * SPDX-License-Identifier: EPL-2.0
+ *  *
+ *  * Contributors:
+ *  *   Broadcom, Inc. - initial API and implementation
+ *
+ */
```

Comment:
Please, check your copyright settings to avoid double asterisks at the beginning